dev_guide/builds.adoc

```diff
@@ -846,43 +850,47 @@ source:
 
 == Binary Source
 
-When the `*BuildConfig.spec.source.type*` is `*Binary*`, the build will expect a
-binary as input, and an inline Dockerfile is optional.
-
-The binary is generally assumed to be a tar, gzipped tar, or zip file depending
-on the strategy. For `*Docker*` builds, this is the build context and an
-optional Dockerfile may be specified to override any Dockerfile in the build
-context. For `*Source*` builds, this is assumed to be an archive as described
-above. For `*Source*` and `*Docker*` builds, if `*binary.asFile*` is set the
-build context will consist of a single file named by the value of `*binary.asFile*`. 
-The `*contextDir*` field may be used when an archive is provided. Custom builds will 
-receive this binary as input on standard input (`stdin`).
+`*Binary*` is a valid value for `*BuildConfig.spec.source.type*`, but is it not something you will
```

Comment:
`but is it not something you will`  *`it is not something`
also `valid value` sounds a bit funky it could be `it is a value that is valid for` or rather in this context `it is a value that could be valid for`